onnx/__init__.py

```diff
@@ -180,59 +197,56 @@ def load_model(
 
 def load_tensor(
     f: Union[IO[bytes], str],
-    format: Optional[Any] = None,  # pylint: disable=redefined-builtin
+    format: _SupportedFormat = "protobuf",  # pylint: disable=redefined-builtin
 ) -> TensorProto:
     """Loads a serialized TensorProto into memory.
 
     Args:
         f: can be a file-like object (has "read" function) or a string containing a file name
-        format: for future use
+        format: The serialization format. Can be one of :py:const:`_SupportedFormat`. Default is "protobuf".
 
     Returns:
         Loaded in-memory TensorProto.
     """
-    s = _load_bytes(f)
-    return load_tensor_from_string(s, format=format)
+    return load_tensor_from_string(_load_bytes(f), format)
 
 
 def load_model_from_string(
     s: bytes,
-    format: Optional[Any] = None,  # pylint: disable=redefined-builtin
+    format: _SupportedFormat = "protobuf",  # pylint: disable=redefined-builtin
 ) -> ModelProto:
     """Loads a binary string (bytes) that contains serialized ModelProto.
 
     Args:
         s: a string, which contains serialized ModelProto
-        format: for future use
+        format: The serialization format. Can be one of :py:const:`_SupportedFormat`. Default is "protobuf".
 
     Returns:
         Loaded in-memory ModelProto.
     """
-    del format  # Unused
-    return _deserialize(s, ModelProto())
+    return _deserialize(s, ModelProto(), format)
 
 
 def load_tensor_from_string(
     s: bytes,
-    format: Optional[Any] = None,  # pylint: disable=redefined-builtin
+    format: _SupportedFormat = "protobuf",  # pylint: disable=redefined-builtin
 ) -> TensorProto:
     """Loads a binary string (bytes) that contains serialized TensorProto.
 
     Args:
         s: a string, which contains serialized TensorProto
-        format: for future use
+        format: The serialization format. Can be one of :py:const:`_SupportedFormat`. Default is "protobuf".
```

Comment:
Any attribute started with `_` is usually left out of the documentation. It should be renamed.

Reply:
I think it will be expanded inline? Let me test that.

Reply:
<img width="767" alt="image" src="https://user-images.githubusercontent.com/11205048/232177502-22c8b33a-de49-4d45-bd16-9403c0f2577a.png">

Looks like the type annotation works. I will remove this duplicated info in the docstring.

Reply:
Done